File: webauthn-server-core/src/main/scala/com/yubico/webauthn/AttestationTrustResolver.scala
package com.yubico.webauthn

import java.util.Optional

import com.yubico.u2f.attestation.Attestation
import com.yubico.webauthn.data.AttestationObject


trait AttestationTrustResolver {

  def resolveTrustAnchor(attestationObject: AttestationObject): Optional[Attestation]

}

File: webauthn-server-core/src/main/java/com/yubico/webauthn/AttestationTrustResolver.java
package com.yubico.webauthn;

import com.yubico.webauthn.attestation.Attestation;
import com.yubico.webauthn.data.AttestationObject;
import java.security.cert.CertificateEncodingException;


interface AttestationTrustResolver {

  Attestation resolveTrustAnchor(AttestationObject attestationObject) throws CertificateEncodingException;

}
